codec-http/src/main/java/io/netty/handler/codec/http/HttpHeaders.java

```diff
@@ -1436,6 +1427,18 @@ public String get(CharSequence name, String defaultValue) {
     public abstract boolean contains(String name);
 
     /**
+     * @deprecated Use {@link #iteratorCharSequence()}.
+     */
+    @Override
+    @Deprecated
+    public abstract Iterator<Entry<String, String>> iterator();
+
+    /**
+     * Get an iterator to traverse over the underlying name/value pairs.
+     */
+    public abstract Iterator<Entry<CharSequence, CharSequence>> iteratorCharSequence();
```

Comment:
I guess I prefer to start with `iterator*` so that it will be more visible in IDEs (and javadocs) which typically list things alphabetically.